README.md

```diff
@@ -154,14 +154,14 @@ In the container, you can run CLI from the source code using:
 go run ./src/main.go help init
 ```
 
-To compile a local binary to `./target/bin/local`, run in the container:
+To compile a local binary to `./target`, run in the container:
 ```
-make build
+make build-local
 ```
 
-To compile a binary for all architectures to `./target/bin`, run in the container:
+To compile a binary for all architectures to `./target`, run in the container:
 ```
-make build-cross
+make build
```

Comment:
upravil som readme, build-local vygeneruje len jednu binary podla aktualnej arch a OS. Nastavenie aby to generoval do zlozky `target/local/bin/` som vynechal pretoze to je zbytocne kodu naviac a navyse tu zlozku potom generuje aj pre CI build(tam som to prave nevedel vypnut).
Napriklad mne binary vygeneruje do `./target/kbc_darwin_amd64/kbc`